src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -122,6 +122,17 @@ class Telemetry : public PluginBase {
         float yaw_rad_s; /**< @brief Yaw angular velocity */
     };
 
+    /**
+     * @brief Ground speed type.
+     *
+     * Metrics typically displayed on a HUD for fixed wing aircraft.
+     */
+    struct VfrHud {
+        float airspeed_m_s; /**< @brief Current indicated airspeed (IAS) in metres/second. */
+        float throttle; /**< @brief Current throttle setting (0 to 100). */
+        float climb_m_s; /**< @brief Current climb rate in metres/second. */
```

Comment:
If it is a climb rate I would call it `climb_rate_m_s`.

Reply:
no problem